packages/nodes-base/nodes/Asana/Asana.node.ts

```diff
@@ -1625,6 +1757,111 @@ export class Asana implements INodeType {
 					},
 				],
 			},
+			// ----------------------------------
+			//         project:update
+			// ----------------------------------
+			{
+				displayName: 'Workspace',
+				name: 'workspace',
+				type: 'options',
+				typeOptions: {
+					loadOptionsMethod: 'getWorkspaces',
+				},
+				displayOptions: {
+					show: {
+						resource: [
+							'project',
+						],
+						operation: [
+							'update',
+						],
+					},
+				},
+				default: '',
+				description: 'The workspace to create the project in',
+			},
+			{
+				displayName: 'Project',
+				name: 'projectId',
+				type: 'options',
+				displayOptions: {
+					show: {
+						resource: [
+							'project',
+						],
+						operation: [
+							'update',
+						],
+					},
+				},
+				default: '',
+				description: 'The project to update info on.',
+			},
+			{
+				displayName: 'Additional Fields',
+				name: 'additionalFields',
+				type: 'collection',
+				displayOptions: {
+					show: {
+						resource: [
+							'project',
+						],
+						operation: [
+							'update',
+						],
+					},
+				},
+				default: {},
+				description: 'Other properties to set',
+				placeholder: 'Add Property',
+				options: [
+					{
+						displayName: 'Color',
+						name: 'color',
+						type: 'string',
+						default: '',
+						description: 'Color of the project.',
+					},
+					{
+						displayName: 'Due On',
+						name: 'due_on',
+						type: 'dateTime',
+						default: '',
+						description: 'The day on which this project is due. This takes a date with format YYYY-MM-DD.\n',
+					},
+					{
+						displayName: 'Name',
+						name: 'name',
+						type: 'string',
+						default: '',
+						description: 'The name of the project.',
+					},
+					{
+						displayName: 'Notes',
+						name: 'notes',
+						type: 'string',
+						default: '',
+						description: 'Basic description or notes for the project.',
+					},
+					{
+						displayName: 'Owner',
+						name: 'owner',
+						type: 'string',
+						default: '',
+						description: 'The new assignee/cardinal for this project.',
+					},
+					{
+						displayName: 'Team',
+						name: 'team',
+						type: 'options',
+						typeOptions: {
+							loadOptionsMethod: 'getTeams',
```

Comment:
Add load dependency. Otherwise, if the workspace is not set, this will not work.

Reply:
Added a load dependency and I added workspace as a general field for it with a getWorkspaces option for it